Apps/phwhois/whois_connector.py

```diff
@@ -151,7 +151,10 @@ def _is_ip(self, input_ip_address):
         ip_address_input = input_ip_address
 
         try:
-            ipaddress.ip_address(unicode(ip_address_input))
```

Comment:
Am I reading this wrong or are there two trys here?

Reply:
Works for py2 and py3 compat